### PR TITLE
Add test cases and fix import error in tests/test.py

### PR DESCRIPTION
**Pull request by todipratik**
Addressed issue #10 

-----
[View rendered README.md](https://github.com/todipratik/git-profiler-bot/blob/master/README.md)

### CODE REVIEW

File: tests/test.py [hunk 1]
```diff
@@ -1,4 +1,54 @@
...
+    assert int(util('Public Repos')) == result['public_repos']
+
+def test_public_gists():
+    assert int(util('Public Gists')) == result['public_gists']
+
+def test_followers():
+    assert int(util('Followers')) == result['followers']
+
+def test_following():
+    assert int(util('Following')) == result['following']
```
Comment (aashutoshrathi) [inline tests/test.py]: A newline at the end is needed.

File: tests/test.py [hunk 2]
```diff
@@ -1,4 +1,54 @@
...
+sys.path.append('../')
 from handler import stalk
+import requests
+import os
 
 # Yeah Testing is cool and totally worth it!
-print(stalk('aashutoshrathi'))
+user = 'aashutoshrathi'
+delimiter = '</strong>'
+os.environ['CONTRI_API'] = 'https://github-contributions-api.herokuapp.com/'
```
Comment (aashutoshrathi) [inline tests/test.py]: Also, can we have one test for CONTRI_API?

Comment (todipratik) [reply tests/test.py]: Can you explain more on this? I can't figure out a way to test this.

Comment (aashutoshrathi) [reply tests/test.py]: We can just check if that's a number of today or not.

File: tests/test.py (hunk 1 above)
Comment (todipratik) [reply tests/test.py]: Added

File: tests/test.py (hunk 2 above)
Comment (todipratik) [reply tests/test.py]: Done

**Comment by todipratik** [comment]
@aashutoshrathi I have addressed the comments.

**Comment by aashutoshrathi** [comment]
@todipratik Can you do it, ASAP?
I want you to get your funds before friday.

**Comment by todipratik** [comment]
Actually while working on it, I was thinking to run tests before every deployment, i.e. auto run tests after running `serverless deploy`, but seems that is not possible.

Running tests from parent directory using `pytest tests/test.py` needs to figured out. Can we take it as a separate task?

**Comment by aashutoshrathi** [comment]
Sure, I'll handle that in separate PR. Can you rebase and add one more test for streaks?

**Comment by aashutoshrathi** [comment]
@todipratik on it?

**Comment by aashutoshrathi** [comment]
Yes, you have to force push after rebase.

**Comment by todipratik** [comment]
Done, please check

**Comment by todipratik** [comment]
Done

**Pull request merged**